packages/integration-tests/projects/suite-web/tests/suite/safety-checks-warning.test.ts

```diff
@@ -29,38 +29,62 @@ describe('safety_checks Warning For PromptTemporarily', () => {
         // Start in the device settings to easily open safety_checks setting modal.
         cy.prefixedVisit('/');
         cy.passThroughInitialRun();
+        cy.discoveryShouldFinish();
         cy.getTestElement('@suite/menu/settings').click();
         cy.getTestElement('@suite/menu/settings-index').click();
         cy.getTestElement('@settings/menu/device').click();
 
         // Set safety_checks to `PromptTemporarily'.
         // TODO: do this via the `applySetting` task once https://github.com/trezor/trezor-user-env/issues/54
         // is resolved.
-        cy.getTestElement('@settings/device/safety-checks-button').click();
-        cy.get(`[data-test="@radio-button-prompt"]`).click();
-        cy.getTestElement('@safety-checks-apply').click();
-        cy.task('pressYes');
```

Comment:
Why do we duplicate this to all the tests?

If that's necessary, please take the comments into account, they're part of the code too.

Reply:
You should create a helper function instead of duplication.